ui/app/AppLayouts/Chat/views/communities/CommunityNewTokenView.qml

```diff
@@ -348,7 +341,14 @@ StatusScrollView {
             text: qsTr("Preview")
             enabled: d.isFullyFilled
 
-            onClicked: root.previewClicked()
+            onClicked: {
+                var trimmedText = descriptionInput.text.replace(/\s+/g, " ");
```

Comment:
```suggestion
                const trimmedText = descriptionInput.text.replace(/\s+/g, " ");
```
It will replace all new lines with just a space. It's probably not the expected behavior here.